generate.js

```diff
@@ -78,11 +78,17 @@ function cli (args) {
   const opts = argv(args)
 
   if (opts._[0] && existsSync(opts._[0])) {
-    log('error', 'directory ' + opts._[0] + ' already exists')
+    if (opts._[0] !== '.' && opts._[0] !== './') {
```

Comment:
I'm not convinced by this change. I think we should be supporting generating the structure in the current folder, if someone really wants to.

Reply:
I did this. if users pass `.` or `./`, it will jump out of judgement and go to Line 91 to begin generating the structure in the current folder.